deployment/config.go

```diff
@@ -112,6 +113,15 @@ type ElasticSearchSettings struct {
 	CreateRole bool
 }
 
+// JobServerSettings contains the necessary data to deploy a job
+// server in a clustered deployment.
```

Comment:
Is it necessary to have a clustered deployment? A job server can be run with a team edition server as well right? Although it will be problematic for sure, but it's still possible right?

Reply:
It's not necessary actually. Technically it's doable so will update the comment.